### PR TITLE
disable transfer button if a measure from family/Set is already trans…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
…ferred.

## JIRA Ticket
[https://jira.cms.gov/browse/MAT-4815](https://jira.cms.gov/browse/MAT-4815)

## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [x] Tests have been run locally and pass.

## Screenshots (if appropriate)
image
image

- [ ] None applicable

### CODE REVIEW

**Review by adongare — DISMISSED** [review]

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java [hunk 1]
```diff
@@ -132,11 +132,12 @@ public void setExportOptionsBasedOnVersion(String releaseVersion, boolean isComp
...
 			vp.add(humanReadableRadio);
 			vp.add(allRadio);
-			// Transfer to Madie is displayed only for the measure owner
-			if (currentMeasureOwnerId != null && MatContext.get().getFeatureFlagStatus(FeatureFlagConstant.MADIE)
-					&& currentMeasureOwnerId.equals(MatContext.get().getCurrentUserInfo().userId)) {
+			if (MatContext.get().getFeatureFlagStatus(FeatureFlagConstant.MADIE)) {
 				vp.add(transferToMadieRadio);
 			}
+			if (!isTransferableToMadie) {
+				transferToMadieRadio.setEnabled(false);
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/measure/ManageMeasureExportView.java:139]: Any concern with altering the enabled state if the radio button isn't displayed?

I'm guessing not.

**Review by jkotanchik-SB — APPROVED** [review]

File: src/main/java/mat/client/export/measure/ManageMeasureExportView.java (hunk 1 above)
Comment (RohitKandimalla) [reply src/main/java/mat/client/export/measure/ManageMeasureExportView.java:139]: I guess not
